sycl/include/CL/sycl/detail/sycl_mem_obj_t.hpp

```diff
@@ -120,7 +120,8 @@ template <typename AllocatorT> class SYCLMemObjT : public SYCLMemObjI {
   }
 
   void releaseHostMem(void *Ptr) override {
-    MAllocator.deallocate(allocator_pointer_t<AllocatorT>(Ptr), get_size());
+    if(Ptr)
```

Comment:
'if(Ptr)' -> 'if (Ptr)'.
Otherwise, looks good to me.

Reply:
Done.